lib/ui5Framework/AbstractResolver.js

```diff
@@ -147,10 +150,38 @@ class AbstractResolver {
 		};
 	}
 
+	/**
+	 *
+	 * @param {string} version
+	 * @param {Object} options
+	 */
+	static async resolveVersion(version, {ui5HomeDir, cwd} = {}) {
+		let spec;
+		if (version === "latest") {
+			spec = "*";
+		} else if (versionRegExp.test(version)) {
+			spec = version;
+		} else {
+			throw new Error(`Framework version specifier "${version}" is incorrect or not supported`);
```

Comment:
```suggestion
			throw new Error(`Framework version specifier "${version}" is incorrect or not supported. Try "1.76.0" or "latest"`);
```

Reply:
Hm, 1.76 will become outdated very soon, so I think that's not a good idea. Maybe we should point to the versionoverview.html instead?

Reply:
I'm more concerned about *how* the version needs to be specified. But you are right, mentioning an old version here might be irritating.

Reply:
Maybe something like `Try "1.xx", "1.xx.x" or "latest"` ?